arxiv/base/config.py

```diff
@@ -156,3 +156,12 @@
 """Analytics tracker cookie domain."""
 ANALYTICS_SITE_ID = os.environ.get("ANALYTICS_SITE_ID", "1")
 """Analytics tracker site ID."""
+
+TRACKBACK_SECRET = os.environ.get("TRACKBACK_SECRET", "baz")
+
+DEFAULT_DB = "sqlite:///../tests/data/browse.db"
+DEFAULT_LATEXML_DB = "sqlite:///..test/data/latexml.db"
+
+CLASSIC_DB_URI = os.environ.get("CLASSIC_DB_URI", DEFAULT_DB)
+LATEXML_DB_URI = os.environ.get("LATEXML_DB_URI", DEFAULT_LATEXML_DB)
+ECHO_SQL = os.environ.get("ECHO_SQL", True)
```

Comment:
not sure we would want the default to be true

Reply:
Fair point